spec/DayDifference.spec.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use johncorrelli\dayDifference;
+use DateTime;
+
+describe('dayDifference', function() {
```

Comment:
`dayDifference` is for test to cover current name of the class. This gets correct in the next commit where the class is refactored and properly named.